recipes/fgap/meta.yaml

```diff
@@ -0,0 +1,24 @@
+about:
+  home: https://github.com/pirovc/fgap
+  license: MIT
+  summary: "FGAP: an automated gap closing tool"
+build:
+  number: 0
+  skip: True  # [osx]
+  
+package:
+  name: fgap
+  version: 1.8.1
+source:
+  fn: e3004998f9058e8505b7812544d4c1dbd595a169.tar.gz
```

Comment:
You can still use 
```yaml
  fn: fgap-1.8.1.tar.gz
```
`conda-build` would default to `e3004998f9058e8505b7812544d4c1dbd595a169.tar.gz` if `fn` isn't given, but we can also use a more meaningful name like `fgap-1.8.1.tar.gz` if we want to.